### DISCUSSION TITLE
Option to implement non-existent function

### DISCUSSION THREAD

**Discussion by cillfore** [godotengine/godot-proposals#8025] [Editor]
Right now, if you call a function that doesn't exist, you'll be hit with the **'Function "function_you_want()" not found in base self.'** message. Of course, the only way around that is to create the function.

It would be a lot quicker for my workflow if I could highlight and right click (or use a keyboard shortcut) on the line I just wrote to "Implement Function"; this would create the func at the bottom of the script, maybe passing the parameters inside as the initial names, with a body of "pass".

**Comment by theraot** [comment]
This seems to be in the same spirit as https://github.com/godotengine/godot-proposals/discussions/8004
